internal/ogc/features/testdata/expected_schema_descr_from_db_snippet.html

```diff
@@ -0,0 +1,528 @@
+                        </td>
+                    </tr>
+                
+            
+                
+
+                
+                
+                
+
+                
+                    <tr>
+                        <td class="text-nowrap">
+                        
+                            status
+                        
+                        </td>
+                        <td class="text-nowrap">string</td>
+                        <td class="text-nowrap">nee</td>
+                        <td class="text-wrap">
+                            
+
+                            
+                        </td>
+                    </tr>
+                
+            
+                
+
+                
+                
+                
+
+                
+                    <tr>
+                        <td class="text-nowrap">
+                        
+                            datum_strt
+                        
+                        </td>
+                        <td class="text-nowrap">string</td>
+                        <td class="text-nowrap">nee</td>
+                        <td class="text-wrap">
+                            
+
+                            
+                        </td>
+                    </tr>
+                
+            
+                
+
+                
+                
+                
+
+                
+                    <tr>
+                        <td class="text-nowrap">
+                        
+                            datum_eind
+                        
+                        </td>
+                        <td class="text-nowrap">string</td>
+                        <td class="text-nowrap">nee</td>
+                        <td class="text-wrap">
+                            
+
+                            
+                        </td>
+                    </tr>
+                
+            
+                
+
+                
+                
+                
+
+                
+                    <tr>
+                        <td class="text-nowrap">
+                        
+                            document
+                        
+                        </td>
+                        <td class="text-nowrap">string</td>
+                        <td class="text-nowrap">nee</td>
+                        <td class="text-wrap">
+                            
+
+                            This is a multi line placeholder that describes the document field.This is a multi line placeholder that describes the document field.This is a multi line placeholder that describes the document field.This is a multi line placeholder that describes the document field.This is a multi line placeholder that describes the document field.This is a multi line placeholder that describes the document field.This is a multi line placeholder that describes the document field.This is a multi line placeholder that describes the document field.
```

Comment:
Mutiline werkt niet helemaal, aangezien de newlines wegvallen. Ik denk dat dit ook niet een use case is die we perse moeten ondersteunen. Lange teksten wel, multiline niet. Je zou de testdata daar nog op kunnen aanpassen.